src/main/java/com/bunq/sdk/model/generated/object/CardCountryPermission.java

```diff
@@ -22,11 +22,29 @@
   @SerializedName("expiry_time")
   private String expiryTime;
 
+  /**
+   * The id of the card country permission entry.
+   */
+  @Expose
+  @SerializedName("id")
+  private Integer id;
+
   public CardCountryPermission(String country) {
     this.country = country;
   }
 
   /**
+   * The id of the card country permission entry.
```

Comment:
I'd` on this line and the one a few lines above, consistency.

Reply:
generated